js/index.js

```diff
@@ -26,4 +28,54 @@ for (let i=0; i < skills.length; i++) {
     skill.innerText = skills[i];
 
     skillsList.appendChild(skill);
-}
\ No newline at end of file
+}
+
+const name = document.querySelector("#name");
+const email = document.querySelector("#email");
+const message = document.querySelector("#message");
+
+
+const messageForm = document.querySelector('#form')
+
+
+messageForm.addEventListener('submit', (event) => {
+
+    event.preventDefault();
+
+    // const nameInput = event.target.name
+    // const emailInput = event.target.email
+    // const messageInput = event.target.message   I couldnt figure out the way the hw meant for me to do
+
+
+    // console.log(nameInput);
+    // console.log(emailInput);
+    // console.log(messageInput);
+    
+    const messageSection = document.querySelector("#messages");
+    
+    const messageList = document.querySelector("#messageslist");
+
+    const newMessage = document.createElement("li");
+    
+    newMessage.innerHTML = `<a href="mailto:${email.value}">${name.value}</a><span> ${message.value} </span>`
+
+    messageList.appendChild(newMessage);
+
+    const removeButton = document.createElement("button");
+
+    removeButton.innerText = "remove";
+
+    removeButton.setAttribute('type','button');
+
+    newMessage.appendChild(removeButton);
+
+    removeButton.addEventListener('click', (event) => {
+
+        messageList.removeChild(newMessage);
+    
+    
+    });
+
+    document.querySelector('#form').reset();
```

Comment:
Since `event.target` points to the `form` you can also do this by doing `event.target.reset()`